vulture/core.py

```diff
@@ -520,6 +530,11 @@ def csv(exclude):
         help='Comma-separated list of paths to ignore (e.g.,'
         ' "*settings.py,docs/*.py"). {glob_help} A PATTERN without globbing'
         ' characters is treated as *PATTERN*.'.format(**locals()))
+    parser.add_argument(
+        '--ignore-decorators', metavar='PATTERNS', type=csv,
+        help='Comma-separated list of decorators - The functions using these'
+        ' decorators are ignored (e.g., "app.route").'
```

Comment:
Comma-separated list of decorator names. Functions and classes using these decorators are ignored (e.g., "@app.route").